scripts/build-backend.sh

```diff
@@ -46,28 +46,38 @@ echo -en "Minifying with noderify..."
   --filter=async_hooks \
   --filter=utf-8-validate \
   --filter=bufferutil \
-  --filter=worker_threads \
   --filter=http2 \
   --filter=pino-pretty \
   --filter=memcpy \
   --filter=diagnostics_channel \
+  --filter=perf_hooks \
   index.js | \
   # Apologies to future contributors: this worker needs to load better-sqlite3,
   # and I could not get bindings() to correctly load the native code when this is
   # not at the project root. If the path name of the worker changes in the
   # original module then this replacement needs to be updated. Apologies for the
   # fragility, but this was the best solution I could find in limited time.
-  sed 's|\./lib/mbtiles_import_worker\.js|../../../../mbtiles_import_worker.js|' > \
+  sed 's|\.\./lib/mbtiles_import_worker\.js|../../../../../mbtiles_import_worker.js|' | \
+  # Additional apologies for this, which is even more fragile. Context is that 
+  # Piscina introduces its own worker file (unfortunately with a very generic name of `worker.js`)
+  # so we need to update the path it references to point to the eventually relocated
+  # file within the nodejs-project directory.
+  sed "s|'worker\.js'|'../../../../piscina_worker.js'|" > \
   ../nodejs-project/index.js
 
 # The worker in MapServer needs to be bundled separately, and we need to move it
 # and search & replace the reference to the file in the main bundle so that all
 # the paths resolve correctly
 "$(npm bin)/noderify" \
   --replace.bindings=bindings-noderify-nodejs-mobile \
-  --filter=worker_threads \
   node_modules/@mapeo/map-server/dist/lib/mbtiles_import_worker.js > \
-  ../nodejs-project/mbtiles_import_worker.js
+  ../nodejs-project/mbtiles_import_worker.js \
+
+# Piscina uses its own worker to initiate other workers, 
+# so we need to bundle it seperately
+"$(npm bin)/noderify" \
+  node_modules/piscina/dist/src/worker.js > \
+  ../nodejs-project/piscina_worker.js
```

Comment:
not entirely sure if there are any bindings i need to replace here - how would i check? ran fine otherwise when testing some endpoints

Reply:
You don't need any replacements. You can check by looking at the source code https://github.com/piscinajs/piscina/tree/current/src and seeing what modules are required.